services/common/config/rs-server.yaml

```diff
@@ -12,75 +12,72 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# THIS FILE WAS AUTOMATICALLY CREATED FROM:
+#  - rs-server/services/common/config/rs-server.template.yaml
+#  - rs-server/resources/sync_config_files.py
+# DON'T MODIFY IT DIRECTLY !
+
 external_data_sources:
-  # AUXIP. For now only one ADGS but in the future there will be another one
   adgs:
-    domain: mockup-auxip-adgs-svc.processing.svc.cluster.local
+    domain: mockup-station-adgs-svc.processing.svc.cluster.local
```

Comment:
@agrosu1978 @Padeanu is this domain value used somewhere ? Here I've taken the values used in the helm charts to be consistant. I guess this value (that describes a pod url in the cluster) has no meaning in rs-server in local mode.

Reply:
I used the configuration provided in the user story 352, please have a look at:
`Default configuration (local mode format. To be converted to Kubernetes secrets for cluster mode):`
The values from the domain is not yet used, but it will probably be:
`one function that returns the configuration for a given URL, by looking at which entry defines a domain that matches the URL domain, or raises an error if not exactly one entry is found`

Reply:
OK so for now we can use any value since this is not used, I suggest to keep the value (in green above) that is defined in the helm charts for now